### ISSUE TITLE
Accessibility issues for web configuration

### ISSUE THREAD

**Issue by klues** [asterics/FLipMouse-esp32#5]
The current web configuration already has several accessibility features (accesskeys, live-regions, hidden headers, ...) and was tested with ChromeVox screen reader in Google Chrome. See the current version at:
https://asterics.github.io/FLipMouse-esp32/index.htm

However these issues still could be improved:
1.  add accessible tooltips for technical terms like "Deadzone", maybe like these https://www.barrierefreies-webdesign.de/knowhow/tooltip/tooltip-mit-aria.html
2. Many inputs on the configuration page directly change the configuration of the FlipMouse (sensitivity, sip/puff levels, slot selection, ...) and do not have any "submit" button. While this may be comfortable for some users it is maybe also suboptimal for users of screen readers. Maybe there should be a `test` and a `save` button like in the old C# configuration GUI? However the comfortable "live" editing of the configuration would be lost.

**Comment by klues** [comment]
also see this link for, WAVE accessibility checker:
http://wave.webaim.org/report#/https://asterics.github.io/FLipMouse-esp32/index.htm
